### ISSUE TITLE
Load blob into "Content-Type": "application/json"

### ISSUE THREAD

**Issue by justinmakaila** [wkh237/react-native-fetch-blob#417]
I'm trying to embed an image from the file system into a request to the Clarifai API. The API requires a base64 encoded data string embedded in a JSON object (spec [here](https://developer.clarifai.com/guide/predict#via-image-bytes) [switch it to cURL]).

For those who didn't click the link, the body should look like this:
```
{
  inputs: [
    {
      data: {
        image: { 
          base64: (base64 file contents)
        }
      }
    }
  ]
}
```
What's the best way to achieve this? Poking through the iOS source, it appears the request is only built if it finds a wrapped file path at the top level of the body, or if it's embedded in a multipart request it must be located at the `data` key...

**Comment by wkh237** [comment]
From my understanding, if the `Content-Type` is not `octet/*` or `*;base64` the request body will be built as-is. 

```js
RNFetchblob.fetch('POST', url, {
  'Content-Type' : 'application/json'
}, JSON.stringify({
  inputs : [
    {
      data : { image : { base64 } }
    }
  ]
}))
```

**Comment by justinmakaila** [comment]
Yeah but I'm not trying to load the base64 into the js context. I want it
to happen in the native context.

**Comment by wkh237** [comment]
It's possible but would be a very tricky way, like combining the use of `fs.appendFile`, `fs.writeFile` and set encoding to `uri`. 

I think we can just propose a new API like `fs.replace` in `0.11.x`

**Comment by justinmakaila** [comment]
I think we can do it another way. I have a PR kind of thought out
